pvlib/irradiance.py

```diff
@@ -489,9 +489,11 @@ def grounddiffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
     The calculation is the last term of equations 3, 4, 7, 8, 10, 11, and 12.
 
     [2] albedos from:
```

Comment:
Please move these comment lines to where SURFACE_ALBEDO is defined, unless that violates some python grammar rule (@wholmgren ??).  You can drop the '[2]'  Thanks again for these changes.

Reply:
Thank you both. I have added inline comments at line 20.